### ISSUE TITLE
If a project is billable any time entry under the project that is marked non-billable must report an warning to project owner

### ISSUE THREAD

**Issue by buep** [Praqma/toggl-rules#14]

**Comment by buep** [comment]
If a project is a billable, there can not be time entries that are non-billable. And the other way around.
All time entries must follow the project on this setting.

**Comment by edipetres** [comment]
Did a bit of an experimentation in 3b0c071 and found that we will need to introduce pagination of requests as toggl only sends back 50 at a time. 

Also, it's not obvious how the time entries are billed, as none of the entry points contained any data at the billable parameter. See example below:
```js
{
   "id":734296816,
   "pid":71947521,
   "tid":null,
   "uid":2972842,
   "description":"trip back to Oslo",
   "start":"2017-11-16T19:00:00+01:00",
   "end":"2017-11-16T21:10:00+01:00",
   "updated":"2017-11-17T16:00:33+01:00",
   "dur":7800000,
   "user":"Sami Alajrami",
   "use_stop":true,
   "client":"Volvo IT",
   "project":"Volvo travel time (non-invoiceable)",
   "project_color":"0",
   "project_hex_color":"#f1c33f",
   "task":null,
   "billable":0,
   "is_billable":true,
   "cur":"EUR",
   "tags":[

   ]
},

```

**Comment by buep** [comment]
hmm, let's talk. I think the data is fine, and I really want the invoice line report not to depend on bill-able flags, but basically any project at a client should give an invoice line if it isn't an Account Management project.
